### PR TITLE
InProgress

### PR DESCRIPTION
**Pull request by jeremystoll**
Completed Tests for your approval.

### CODE REVIEW

File: tests/string_segmenter_test.rb [hunk 1]
```diff
@@ -2,7 +2,20 @@
 require 'string_segmenter'
 
 class StringSegmenterTest < Minitest::Test
-  # Your test methods go here.
+  def test_1
+    string = "diamondbluecrystalquantitygorilla"
+    assert_equal("diamond blue crystal quantity gorilla", segment_string(string))
```
Comment (sumeetjain) [inline tests/string_segmenter_test.rb]: The method you will write should return an _array_.

Comment (jeremystoll) [inline tests/string_segmenter_test.rb]: Fixed, I hope.

**Comment by sumeetjain** [comment]
:+1:

**Comment by sumeetjain** [comment]
:+1:

**Pull request closed without merging**